### PR TITLE
CP-41119:Remove Citrix.Repo from xe-guest-utilities

### PR DESCRIPTION
**Pull request by MinghuiHu**
Signed-off-by: t_minghuih <minghui.hu@citrix.com>

### CODE REVIEW

File: mk/Citrix.repo [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [citrix]
```
Comment (liulinC) [inline mk/Citrix.repo:1]: The repo ID and repo filename are still Citrix.

File: mk/citrix.list [hunk 2]
```diff
@@ -1,2 +1,2 @@
-deb     http://updates.vmd.citrix.com/@PRODUCT_BRAND@/@PRODUCT_VERSION@/debian/ ^DISTRO^ main
```
Comment (liulinC) [inline mk/citrix.list:1]: is the `xensource` ready now?

File: mk/Makefile.deb [hunk 3]
```diff
@@ -52,11 +52,9 @@ SOURCES += $(SOURCEDIR)/xe-linux-distribution
 SOURCES += $(SOURCEDIR)/xe-daemon
 SOURCES += $(SOURCEDIR)/xenstore
 SOURCES += $(SOURCEDIR)/LICENSE
-SOURCES += $(SOURCEDIR)/citrix.list
 SOURCES += $(SOURCEDIR)/xen-vcpu-hotplug.rules
 SOURCES += $(DEBIANDIR)/xe-linux-distribution.init
 SOURCES += $(DEBIANDIR)/xe-guest-utilities.postinst
-SOURCES += $(DEBIANDIR)/xe-guest-utilities.prerm
```
Comment (liulinC) [inline mk/Makefile.deb:59]: why this file is excluded?

File: mk/xe-guest-utilities.spec.in [hunk 4]
```diff
@@ -105,7 +96,7 @@ fi
 if [ $1 -eq 0 ] ; then
     service xe-linux-distribution stop >/dev/null 2>&1
     /sbin/chkconfig --del xe-linux-distribution >/dev/null
-    rm -f /etc/yum.repos.d/Citrix.repo || /bin/true
+    /bin/true
```
Comment (liulinC) [inline mk/xe-guest-utilities.spec.in:99]: no need to just keep `/bin/true`, you can just remove it

**Review by liulinC — COMMENTED** [review]
Please also update your commit message to clarify the `Citrix repo` for package manager sources, as the repo may be misunderstand to code repo.

File: mk/Makefile.deb (hunk 3 above)
Comment (MinghuiHu) [reply mk/Makefile.deb:59]: There's only one command `rm /etc/apt/sources.list.d/citrix.list`, so I just removed this whole file.

File: mk/citrix.list (hunk 2 above)
Comment (MinghuiHu) [reply mk/citrix.list:1]: This feature is not yet implemented and needs to be discussed. I have to confirm whether citrix.list and citrix.repo still need to exist. If not I can remove these two.

**Comment by stormi** [comment]
Related issue: https://github.com/xenserver/xe-guest-utilities/issues/95

**Review by xihuan-citrix — APPROVED** [review]

**Comment by xihuan-citrix** [comment]
Hi @MinghuiHu , could your test result, e.g. the build result actually not exist the removed contents?

**Comment by MinghuiHu** [comment]
> Hi @MinghuiHu , could your test result, e.g. the build result actually not exist the removed contents?

After manually make build, I grep everything related to citrix repo

**Review by liulinC — APPROVED** [review]

**Pull request merged**